src/Features/VisualBasic/Portable/InitializeParameter/VisualBasicAddParameterCheckCodeRefactoringProvider.vb

```diff
@@ -54,5 +54,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
         Protected Overrides Function EscapeResourceString(input As String) As String
             Return input.Replace("""", """""")
         End Function
+
+        Protected Overrides Function CreateParameterCheckIfStatement(options As DocumentOptionSet, condition As ExpressionSyntax, ifTrueStatement As StatementSyntax) As StatementSyntax
+            Return SyntaxFactory.MultiLineIfBlock(
```

Comment:
💭 Technically we could do the same thing here, but I'm not sure it's common enough to need to. I'd be fine leaving it as a follow-up, especially considering `AllowEmbeddedStatementsOnSameLine` would need to be moved to the shared options first.

Reply:
agreed.

fwiw, we don't check AllowEmbeddedStatementsOnSameLine  in vb at all.  it was really intended for the C# case (and may just be a c# option currently).  we have very little use of this pattern in vb.